.github/workflows/draft-release.yaml

```diff
@@ -0,0 +1,26 @@
+on:
+  push:
+    # Sequence of patterns matched against refs/tags
+    tags:
+    - 'v*' # Push events to matching v*, i.e. v1.0, v20.15.10
+    - 'protocol*' # Push events to matching v*, i.e. v1.0, v20.15.10
+    
+name: Create Draft Releases
+
+jobs:
+
+  release-tags:
+    name: draft-release
+
+    steps:
+
+    - name: Create Draft Release
+      id: create_draft_release
+      uses: actions/create-release@v1
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      with:
+        tag_name: ${{ github.ref }}
+        release_name: Release ${{ github.ref }}
+        draft: true
```

Comment:
draft release? is it valid?

Reply:
I wanted to have a run with this as a draft before trusting it, it is valid.

Reply:
Ok cool, maybe we should understand how to make it configurable somehow

Reply:
like some fancy regex depending on the tag

Reply:
but you can do it as followup after we enable it for real releases :)